src/main/java/com/github/hehe3301/bot/CommandHandler.java

```diff
@@ -105,8 +105,25 @@
         commandMap.put("time", (event, args) ->
         {
             //TODO implement
-            String reply = event.getAuthor().mention() + "\n"
-                    + "This feature has not been implemented yet!";
+            String reply = event.getAuthor().mention() + "\n";
+            switch(args.size())
```

Comment:
This is a big of an ugly switch statement. Consider making translateTime variadic instead of having all these different signatures.

Reply:
Future Fix

Reply:
Allowed because it works for now